aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -128,6 +243,17 @@ module aptos_std::ristretto255_bulletproofs {
         proof: vector<u8>,
         num_bits: u64,
         dst: vector<u8>): bool;
+    
+    /// Aborts with `error::invalid_argument(E_RANGE_NOT_SUPPORTED)` if an unsupported `num_bits` is provided.
```

Comment:
For consistency with the previous native for individual (unbatched) Bulletproof verification, please make it so that this aborts with `E_DESERIALIZE_RANGE_PROOF` when the deserialization fails.

Plus, this would also give more info to the caller about why the call failed.

Reply:
Done